### DISCUSSION TITLE
mania scroll speed slider should be logarithmic

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#21130] [Ideas] [answered]
currently there is like 10 pixels of room between scroll speed 40, and scroll speed 20, and the other 95% of the slider is dedicated to slow scroll speeds. This should be changed to have more detail for the higher speeds, and lower detail for the slower speeds.
Unless my brain is fucked this would be logarithmic.

**Comment by smoogipoo** [comment]
I believe it used to be logarithmic at some point, but we changed it to the osu!-stable 1-40 scale because it was making people confused. That's going to be the biggest hurdle to solve here, and perhaps automatic import of osu-stable config will help with that.

**Reply by Doksha** [reply]
What do you mean by that? if it moved to the 1-40 scale, shouldnt the slider at least be linear? currently it is exponential in terms of the 1-40 system.
If it used to be a different system, it was likely the exact other way around, since it is currently based off ms, not number

**Comment by Xinnoh** [comment]
Part of the reason why this is an issue is because the range of speeds is quiet large.
osu's is 10 seconds which is in the realm of unreadable even for 1* maps.
Chunithm's min scroll speed is 5 seconds which is a good benchmark, changing osu's min speed to 5s would help bandaid this issue a bit if nothing else is applied.

Try playing any 1* map at a speed slower than 5s before suggesting why 10s should be kept.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23691
